### PR TITLE
LOG-5028: Replace lua transforms with VRL

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* Replaces all lua transforms with VRL equivalent
* Adds linter to check use of lua transforms

### Links
https://issues.redhat.com/browse/LOG-5028

### CODE REVIEW

File: internal/generator/vector/normalize/dedot_labels.go [hunk 1]
```diff
@@ -2,64 +2,38 @@ package normalize
...
 import (
 	. "github.com/openshift/cluster-logging-operator/internal/generator/framework"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/elements"
 	"github.com/openshift/cluster-logging-operator/internal/generator/vector/helpers"
+	"strings"
 )
 
-// Dedotting namespace labels
-// Replaces '.' and '/' with '_'
+const (
```
Comment (vparfonov) [inline internal/generator/vector/normalize/dedot_labels.go:10]: nit: what about using go:embed feature and put this script to dedot.vrl file

Comment (jcantrill) [reply internal/generator/vector/normalize/dedot_labels.go:10]: :shrug: We could, seems like an extra layer here

**Pull request merged**